### ISSUE TITLE
func-call-spacing does not recognize all line terminators

### ISSUE THREAD

**Issue by fasttime** [eslint/eslint#7923]
**Tell us about your environment**

* **ESLint Version:** 3.13.1
* **Node Version:** 7.4.0
* **npm Version:** 4.0.5

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```
{ }
```

**What did you do? Please include the actual source code causing the issue.**

```js
/*eslint func-call-spacing: ["error", "always"]*/

fn
();
```

**What did you expect to happen?**

In the snippet above, when the line terminator character between `fn` and `()` is one of `"\r"`, `"\u2028"` or `"\u2029"`, an error should be reported.

**What actually happened? Please include the actual, raw output from ESLint.**

An error is only reported when the line terminator character between `fn` and `()` is `"\n"`.

**Comment by fasttime** [comment]
I've been working on a clone of this rule yesterday and I can submit a patch if desired.

**Comment by platinumazure** [comment]
@fasttime I just need to figure out an easy way to repro so I can mark the bug accepted, but yes, a patch would be welcome!

**Comment by platinumazure** [comment]
I got lazy and just wrote some more tests, and I can confirm this bug.

@fasttime You are welcome to submit a PR. Please let us know if you end up not having time. Thanks!

**Comment by fasttime** [comment]
@platinumazure Thanks! Any help on how to sign the CLA? Or should I sumbit a PR first?

**Comment by platinumazure** [comment]
You can sign the CLA by visiting this link: https://js.foundation/CLA/

Make sure your name and email address that you use to sign the CLA match
the name and email address you put on your commits. (Use git config
user.name and got config user.email to check those in your local Git
instance.) I'm not 100% sure both need to match, but it doesn't hurt.

If you decide to write a PR before signing the CLA, that will work too! In
that case, a comment will be left on the PR saying the CLA needs to be
signed, and you can click on the link there to get started. Just be careful
about making sure you sign with the same name and email as what is in your
commit(s).

It's possible but tedious to change a commit's name/email after pushing it
for a PR, so I recommend signing the CLA first (and checking your Git
configuration as part of that). But ultimately, it's up to you.

Please visit https://gitter.im/eslint/eslint if you need help, and someone
on the team can help you out.

**Comment by fasttime** [comment]
@platinumazure Ok, thanks. The link above took me to a page with nothing but legal notes, but after submitting the PR, I was pointed to https://cla.js.foundation/eslint/eslint which is where I was able to sign the CLA. I'll have time later to fix the commit message.
